src/sys.c

```diff
@@ -724,6 +726,15 @@ JL_DLLEXPORT void jl_raise_debugger(void)
 #endif // _OS_WINDOWS_
 }
 
+JL_DLLEXPORT uint32_t jl_get_LLVM_VERSION(void)
```

Comment:
I don't think we want this here, now that libjulia can be built with a stub no-LLVM version of codegen